docs/reference/rack-parameters.md

```diff
@@ -144,6 +144,21 @@ Set this value to the hostname (or IP address) and port number of an HTTP proxy
 
 For more information, see [HTTP Proxy Configuration](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/http_proxy_config.html) in the AWS docs.
 
+#### Attention
+
+The `HttpProxy`param will not change how the apps access the internet, the traffic won't go through the proxy. Only the instances requests will use the proxy for outbound connections, if you want your apps to use the proxy, you have to configure it on the convox.yml:
+
+```
+services:
+  web:
+    build: .
+    port: 3000
+    environment:
+      - http_proxy=10.0.1.124:8888
```

Comment:
I think we should also add HTTP_PROXY/HTTPS_PROXY as it's usually the standard format that third-party libraries use to pick it up. We can keep the lowercase too as some tools rely on it.